docs/sources/reference/components/otelcol/otelcol.processor.probabilistic_sampler.md

```diff
@@ -32,14 +48,22 @@ otelcol.processor.probabilistic_sampler "LABEL" {
-`attribute_source`    | `string`  | Defines where to look for the attribute in `from_attribute`.                                                         | `"traceID"` | no
-`from_attribute`      | `string`  | The name of a log record attribute used for sampling purposes.                                                       | `""`        | no
-`sampling_priority`   | `string`  | The name of a log record attribute used to set a different sampling priority from the `sampling_percentage` setting. | `""`        | no
+Name                  | Type      | Description                                                                                                          | Default          | Required
+----------------------|-----------|----------------------------------------------------------------------------------------------------------------------|------------------|---------
+`mode`                | `string`  | Sampling mode.                                                                                                       | `"proportional"` | no
+`hash_seed`           | `uint32`  | An integer used to compute the hash algorithm.                                                                       | `0`              | no
+`sampling_percentage` | `float32` | Percentage of traces or logs sampled.                                                                                | `0`              | no
+`sampling_precision`  | `int`     | The number of hexadecimal digits used to encode the sampling threshold.                                              | `4`              | no
+`fail_closed`         | `bool`    | Whether to reject items with sampling-related errors.                                                                | `true`           | no
+`attribute_source`    | `string`  | Defines where to look for the attribute in `from_attribute`.                                                         | `"traceID"`      | no
+`from_attribute`      | `string`  | The name of a log record attribute used for sampling purposes.                                                       | `""`             | no
+`sampling_priority`   | `string`  | The name of a log record attribute used to set a different sampling priority from the `sampling_percentage` setting. | `""`             | no
+
+You can set `mode` to `"proportional"`, `"equalizing"`, or `"hash_seed"`.
+The default is `"proportional"` unless either `hash_seed` is configured or `attribute_source` is set to `record`.
+For more information on modes, refer to the upstream Collector's [Mode Selection documentation][mode-selection-upstream] section.
+
+[mode-selection-upstream]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/{{< param "OTEL_VERSION" >}}/processor/probabilisticsamplerprocessor/README.md#mode-selection
```

Comment:
@wildum @clayton-cornell Regarding a [recent discussion](https://github.com/grafana/alloy/pull/1362#discussion_r1695087198), would you be happy with this link?